.circleci/config.yml

```diff
@@ -1,24 +1,24 @@
 # config file for circleci
```

Comment:
Could you update `prettier` to do 2 spaces for yaml/yml files? If it's not already in `master` for this repo, it's in `master` for the other one so you can copy-paste :)

Reply:
Can I just delete the "**.prettier.json**" file and add a "**.prettierrc**" file from the `covid-vaccine-scrapers`?